src/frontend/apps/lti_site/components/VideoWidgetProvider/widgets/UploadSubtitles/ToggleSubtitlesAsTranscript/index.spec.tsx

```diff
@@ -176,14 +176,13 @@ describe('<ToggleSubtitlesAsTranscript />', () => {
       ...mockedVideoSubAsTrans,
       should_use_subtitle_as_transcript: false,
     });
-
-    render(wrapInVideo(<ToggleSubtitlesAsTranscript />, mockedVideoSubAsTrans));
-
     // Update the state with a timed text track in subtitle mode
     act(() => {
       useTimedTextTrack.getState().addResource(mockedTimedTextTrackSubtitle);
     });
 
+    render(wrapInVideo(<ToggleSubtitlesAsTranscript />, mockedVideoSubAsTrans));
```

Comment:
If you put the render after `useTimedTextTrack.ge...`, you don't need to wrap `useTimedTextTrack` inside an act.

By changing the order, be aware that you are not testing the timedTextTracks update state anymore.